### PR TITLE
1930: create vaccines from CDC XML data

### PR DESCRIPTION
**Pull request by kpethtel**
## Description of change
Adds a new job to the scheduler that will create vaccine records based on data from the CDC. This improves upon the current code by ensuring that any changes added by the CDC will automatically be pulled into the VA mobile app.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#1930
https://app.zenhub.com/workspaces/va-mobile-eng-60f1a34998bc75000f2a489f/issues/department-of-veterans-affairs/va-mobile-app/1930

## Things to know about this PR
Data migration was performed here: https://github.com/department-of-veterans-affairs/vets-api/pull/8516

After this is merged, we will release another PR to change the immunizations model to use the new vaccine model.

This has been tested locally by running the sidekiq job and creating the vaccine records.

### CODE REVIEW

File: modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb
Comment (kreek) [inline modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb:8]: The default number of sidekiq retry attempts is 25 (which with its [exponential back-off algorithm](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry) ends up being 21 days) since this is a daily job I think 3-10 would keep it within one day, but we probably only need the lower end.
```ruby
sidekiq_options retry: 3
```

**Review by kreek — APPROVED** [review]
One minor comment about retrying, but LGTM 🚀

File: modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb
Comment (kpethtel) [reply modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb:9]: That's a good point. Added.

File: modules/mobile/spec/workers/vaccines_updater_job_spec.rb
Comment (kreek) [inline modules/mobile/spec/workers/vaccines_updater_job_spec.rb:52]: When you have multiple expectations in one block you can [aggregate failures](https://justincypret.com/blog/aggregate-failures-in-rspec) to avoid one expectation causing the whole block to fail. This matters more when you're working/TDD'ing through a pass-fail cycle locally, but it also makes the merge check report more accurate should it fail.

**Review by kreek — APPROVED** [review]
One comment, but LGTM, feel free to merge once that's in 🚀

File: modules/mobile/spec/workers/vaccines_updater_job_spec.rb
Comment (kpethtel) [reply modules/mobile/spec/workers/vaccines_updater_job_spec.rb]: Done. Apparently I need another review after adding a commit.

**Review by kreek — APPROVED** [review]
LGTM 🚀

**Comment by holdenhinkle** [comment]
Hey @kpethtel,

I just merged master into a pr and saw your update to `config/sidekiq_scheduler.yml`. I noticed that you indented line 277:

```
  Mobile::V0::VaccinesUpdaterJob:

```

I guessing that job won't run because of that indentation.

Anyway, just wanted to let you know. I noticed because I'm trying to added another job to the file and I had to accept your incoming changes.

Holden

**Pull request merged**
